### PR TITLE
chore: override storybook's styling

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/styles/src/index.css [hunk 1]
```diff
@@ -100,7 +162,7 @@
 
   /* shadows */
   --shadow-0: 0 0 rgba(0, 0, 0, 0);
-  --shadow-1: 0 1px 3px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(0, 0, 0, 0.24);
+  --shadow-1: 0 1px 3px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(250, 245, 245, 0.24);
```
Comment (chrisolsen) [inline libs/styles/src/index.css:165]: not sure how this got changed...

**Review by kevingauld — APPROVED** [review]

**Pull request merged**